### ISSUE TITLE
FR: A way to use 10ten as a dictionary

### ISSUE THREAD

**Issue by Dorako** [birchill/10ten-ja-reader#932]
When I'm reading manga online and I see some vocab I'd like to look up, I can't simply hover it - but I'd still like to use 10ten rather than sites like jisho or jotoba. It would be great if we could keybind (and/or click the puck somehow) to get a text field to just feed text into.

**Comment by SaltfishAmi** [comment]
see also: #285

**Comment by Dorako** [comment]
> see also: #285

Ah, I had tried to find exiting issues with keywords like "search", but I should have tried lookup as well.

**Comment by birtles** [comment]
Yes, I want to do this too. There is a concrete plan to add this feature this year but it's still quite a way off.

If you just want to look up like jisho, there is https://hikibiki.app which uses the same data as 10ten and also runs offline. In fact, it currently only runs offline so you will have to wait for it to download the data after opening it the first time.
